a/asusd/monitoring.yaml

```diff
@@ -0,0 +1,5 @@
+releases:
+  id: 376375
+  rss: ~
```

Comment:
Since this is a Gitlab-hosted project you can grab the RSS feed URL from the tags page: https://gitlab.com/asus-linux/asusctl/-/tags